YouTube.js

```diff
@@ -136,8 +147,26 @@ var testCases = [
 				"libraryCatalog": "YouTube",
 				"runningTime": "2:51",
 				"url": "https://www.youtube.com/watch?v=pq94aBrc0pY",
+				"callNumber": "pq94aBrc0pY",
 				"attachments": [],
-				"tags": [],
+				"tags": [
+					"2.0",
+					"Center",
+					"George",
+					"History",
+					"Mason",
+					"Media",
+					"Mozilia",
+					"New",
+					"Research",
+					"University",
+					"Web",
+					"Zotero",
+					"and",
```

Comment:
These are a little bit junk, aren't they? My impression is that YT tags are mostly there for SEO, and we usually try to exclude SEO-focused metadata from being imported.

Reply:
Depends on the video. I saw other entries with only a handful of tags, but only updated the already existing test case.

So should I remove the part that fetches the tags?

Reply:
Yeah, I think it should be removed.